content/documentation/tutorials/nats-kubernetes.md

```diff
@@ -0,0 +1,21 @@
++++
+date = "2017-09-07"
+title = "Using NATS with Kubernetes"
+description = ""
+category = "tutorials"
+[menu.main]
+  name = "Using NATS with Kubernetes"
+  weight = 4
+  identifier = "tutorial-nats-kubernetes"
+  parent = "Tutorials"
++++
+
+# Using NATS with Kubernetes
+
+NATS and Kubernetes are extremely popular together. Both are cloud native systems designed to handle modern, distributed workloads. There are a variety of methods you can use to run NATS and Kubernetes - including many examples from the broader developer community. We are going to take a look at some of those below.
+
+## Running a NATS Cluster on top of Kubernetes
+One of the core contributors to Kubernetes (Paolo Pires) has also been a long-time user and contributor to NATS. Thankfullly, a lot of his work has made it easier to run NATS on Kubernetes. You can follow his excellent Readme, [here](https://github.com/pires/kubernetes-nats-cluster) to get going - just pay attention to the pre-requisites, and make sure you run the latest version(s) of NATS and Kubernetes.
```

Comment:
It may be nice to have Paolo's name link to his GH user profile, `([Paolo Pires](https://github.com/pires))`?